chart/epinio-installer/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: epinio-installer
-version: 0.4.0
+version: 0.5.0
```

Comment:
Done by `updatecli`, has to be removed. Sorry my first thought was false...

Reply:
done